app/views/layouts/application.html.haml

```diff
@@ -0,0 +1,48 @@
+!!! 5
+%html(lang="en")
+  %head
+    %meta(charset="utf-8")
+    %meta(http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1")
+    %meta(name="viewport" content="width=device-width, initial-scale=1.0")
+    %title= content_for?(:title) ? yield(:title) : "Unitern"
```

Comment:
Don't need to do this now but later on there is a gem you can install that will manage the title tags and all the meta tags that you need to add. https://github.com/kpumuk/meta-tags